scripts/cleanOldStats.ts

```
@@ -42,10 +48,66 @@ async function cleanOldStats() {
     const db = await getDb();
+
+    await cleanOldStats(db).catch(console.error); 
+    await cleanDeadWorkers(db).catch(console.error);
+  } finally {
     await db.destroy();
   }
 }
 
-cleanOldStats().catch(console.error); 
\ No newline at end of file
+main()
```

ESLint

[error] Insert `;`

(prettier/prettier)